WPFClient/Communicator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Client;
+using CorEstafette;
+
+namespace WPFClient
+{
+    class Communicator
+    {
+        private readonly HubConnection connection;
+        private readonly List<string> subscribedTopics;
+
+        public Communicator( )
+        {
+            subscribedTopics = new List<string>();
+            connection = new HubConnectionBuilder()
+                .WithUrl("https://localhost:44392/testhub")
+                .Build();
+
+            connection.StartAsync();
+            connection.Closed += async (error) =>
+            {
+                await Task.Delay(new Random().Next(0, 5) * 1000);
+                await connection.StartAsync();
+            };
+        }
+
+        public async Task<string> SubscribeAsync(string topic)
```

Comment:
Subscribe and unsubscribe will need to have another parameter for callbacks, and the communicator needs to cache them locally. When a message comes to that registered topic, the communicator will need to load the cached callback for that topic and invoke it.